EasyReflectometry/sample/elements/materials/material_solvated.py

```diff
@@ -53,24 +117,32 @@ def solvent(self, new_solvent: Material) -> None:
 
         :param new_solvent: Solvent to be used.
         """
-        # Should set the property
         self.material_b = new_solvent
 
     @property
-    def solvent_surface_coverage(self) -> Parameter:
-        """Get the fraction of layer not covered by the material."""
+    def solvent_fraction(self) -> Parameter:
+        """Get the fraction of layer described by the solvent.
+        This might be fraction of:
+        Solvation where solvent is within the layer
+        Patches of solvent in the layer where no material is present.
```

Comment:
Suggested rewording:
`Get the fraction of the layer occupied by the solvent. This could be a result of either water solvating the layer, or incomplete surface coverage of the layer.`